C/c4Certificate.cc

```diff
@@ -173,6 +170,7 @@ Retained<C4Cert> C4Cert::requestFromData(slice certRequestData) {
 
 bool C4Cert::isSigned() { return _impl->isSigned(); }
 
+// NOLINTBEGIN(readability-convert-member-functions-to-static)
```

Comment:
Format: I have seen that there is always an extra empty line after the comment somewhere else.